showkase-processor-testing/src/test/java/com/airbnb/android/showkase_processor_testing/AnnotationCustomizationsTest.kt

```diff
@@ -0,0 +1,57 @@
+package com.airbnb.android.showkase_processor_testing
+
+import com.airbnb.android.showkase.annotation.ShowkaseComposable
+import org.junit.Test
+
+class AnnotationCustomizationsTest: BaseProcessorTest() {
+
+    @Test
+    fun `custom annotation with standard fields`() {
+        val options = mutableMapOf<String, String>()
+        options[ShowkaseComposable.AnnotationOverride] = "ShowkaseProcessorTest.custom_annotation_with_standard_fields.input.CustomAnnotation"
+        compileInputsAndVerifyOutputs(options = options)
+    }
+
+    @Test
+    fun `custom annotation with custom fields`() {
+        val options = mutableMapOf<String, String>()
+        options[ShowkaseComposable.AnnotationOverride] = "ShowkaseProcessorTest.custom_annotation_with_custom_fields.input.CustomAnnotation"
+        options[ShowkaseComposable.FieldOverrideName] = "label"
+        options[ShowkaseComposable.FieldOverrideGroup] = "category"
+        options[ShowkaseComposable.FieldOverrideStyleName] = "style"
+        // This is only supported in KSP at the moment
+        compileInputsAndVerifyOutputs(options = options, modes = listOf(Mode.KSP))
```

Comment:
I'm not really sure why the field overrides only work on KSP at the moment. I didn't have a use for kapt so I didn't spend much time investigating it.